### DISCUSSION TITLE
Renewal lets encript does not work

### DISCUSSION THREAD

**Discussion by mascalzone** [AzuraCast/AzuraCast#5453] [General]
Hello everybody. I wanted to ask if any of you can help me. On June 2nd my ssl certificate on the domain of my azura cast radio installed by doker compose expired.
It was not renewed automatically and I did not receive any notification emails. When I tried to renew it manually, the operation went well, and even after restarting the radio many times, the https does not work.

I also tried with:
`docker-compose exec nginx_proxy_letsencrypt sh`
`./force_renew`

ma ottengo questo:
```
/app # ./force_renew
Creating/renewal cobraradio.it certificates... (myradio.it)
[Thu Jun  2 23:07:22 UTC 2022] Using CA: https://acme-v02.api.letsencrypt.org/directory
[Thu Jun  2 23:07:22 UTC 2022] Creating domain key
[Thu Jun  2 23:07:22 UTC 2022] The domain key is here: /etc/acme.sh/myemail@gmail.com/myradio.it/myradio.it.key
[Thu Jun  2 23:07:22 UTC 2022] Single domain='cobraradio.it'
[Thu Jun  2 23:07:23 UTC 2022] Getting domain auth token for each domain
[Thu Jun  2 23:07:25 UTC 2022] Getting webroot for domain='myradio.it'
[Thu Jun  2 23:07:25 UTC 2022] Verifying: myradio.it
[Thu Jun  2 23:07:28 UTC 2022] myradio.it:Verify error:2001:4b78:1001::501: Invalid response from http://myradio.it/.well-known/acme-challenge/XfHxTbYFkpnv7u4m9ZYVSmmPcjQuCn1Hfvr3LHajlzM: 404
[Thu Jun  2 23:07:28 UTC 2022] Please check log file for more details: /dev/null
/app #

can someone help me?

**Comment by ghost** [comment]
I went to cobraradio.it and after looking at the SSL cert it did confirm that it will expire at days end.
was there a reason you forced the renew? 
did you try doing an update? 
did you update your server before or after you tried to renew the SSL cert?
I did accept the ssl and listened to a few minutes of your radio station

**Reply by ghost** [reply]
I had one more thought that might fix the issue,
make a back up of the certs or rename them and try install a new cert

**Reply by mascalzone** [reply]
Even if the certificate expires at the end of the day it was already non-functional. Indeed cobraradio.it. it was already in http.
I did an apt update and apt upgrade to the ubuntu16 vps after "updating" the certificate.
I followed the guide on azuraccast.

I don't know in which directory to find the certificate files. Could you tell me the path, so I try to rename them and try a new install.

**Reply by ghost** [reply]
start looking in /var/lib/docker/volume

**Reply by mascalzone** [reply]
![Immagine 2022-06-03 195429](https://user-images.githubusercontent.com/19390561/171920330-61abb31d-ef70-416f-8a76-ec94d5f58623.jpeg)
![Immagine 2022-06-03 195721](https://user-images.githubusercontent.com/19390561/171920359-1aee6567-0b1c-4d9d-b5cb-a13f1fafd274.jpeg)

How can I find out which file to name?

**Reply by mascalzone** [reply]
I don't understand. I've tried everything. I can't get it to work in https. I am seriously considering reinstalling all over again. For some reason the certificate is not renewed.
Is there a way to clear all lets-encrypt data in order to retry a clean installation of a new certificate from scratch? If this solution does not exist then perhaps it is better to reinstall azuracast from scratch. In this case I would need a guide on how to uninstall everything cleanly and then start with the new installation.
